test/e2e/e2e_test.go

```diff
@@ -296,8 +387,193 @@ var _ = Describe("Manager", Ordered, func() {
 		//    strings.ToLower(<Kind>),
 		// ))
 	})
+
+	Context("N pipelines complete successfully", func() {
```

Comment:
I think we should mark this as `Ordered` (cf. https://onsi.github.io/ginkgo/#ordered-containers)

> Ordered is a decorator that allows you to mark a container as ordered.  Specs in the container will always run in the order they appear.
They will never be randomized and they will never run in parallel with one another, though they may run in parallel with other specs.


```suggestion
	Context("N pipelines complete successfully", Ordered, func() {
```